docs/src/content/mapping/lodash/random.md

```diff
@@ -0,0 +1,15 @@
+---
+category: Number
+remeda: randomInt
+---
+
+- In Lodash, the `random` function supports generating either a random floating-point number or a random integer. In Remeda, only random integers are supported.
+- In Lodash, the `random` function supports accepting a single argument, which is the upper bound of the random number. In Remeda, the `randomInt` function requires both the lower and upper bounds as arguments.
+
+```ts
+// Lodash
+random(1, 10);
+
+// Remeda
+randomInt(1, 10);
+```
```

Comment:
We should add examples of each unique way our function differs from Lodash, this means we need to give an example where: `_.random()` and `_.random(max)`, not just `_.random(min, max)`, and give a solution for those that need the floating-point variant. You can link to this for that case: https://youmightnotneed.com/lodash#random

Reply:
Good point

Are we gonna have a `randomFloat` at some point?
because for now I'm referencing to this link as a solution